source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -1,33 +1,38 @@
--------------
+What are vendor dependencies?
+-----------------------------
 
-In support of this effort NI (for LabVIEW) and FIRST/WPI (for Java/C++) have developed mechanisms that should make it easy for vendors to plug their code into the WPILib software and for teams to use that code once it has been installed. A brief description of how the system works for each language can be found below.
+A vendor dependency is a way for vendors like CTRE, and REV to add their own custom code that interfaces with their motor controllers and other devices. This way you can interact with your devices via CAN and have access to more complex and in-depth features than traditional PWM control.
 
-The Mechanism - Java/C++
-^^^^^^^^^^^^^^^^^^^^^^^^
+Managing Vendor Dependencies
+----------------------------
+
+Vendor dependencies are installed on a per-project basis (so each robot project can have its own set of vendor dependencies). Vendor dependencies can be installed "online" or "offline". The offline installer will usually be provided by the vendor themselves if they have one and will usually come with additional debugging/utility programs.
 
-For Java and C++ a JSON file describing the vendor library is installed on your system to ``~/wpilib/YYYY/vendordeps`` (where YYYY is the year and ~ is ``C:\Users\Public`` on Windows). This can either be done by an offline installer or the file can be fetched from an online location using the menu item in Visual Studio Code. This file is then used from VS Code to add to the library to each individual project. Vendor library information is managed on a per-project basis to make sure that a project is always pointing to a consistent version of a given vendor library. The libraries themselves are placed in the Maven cache at ``C:\Users\Public\wpilib\YYYY\maven``. Vendors can place a local copy here with an offline installer (recommended) or require users to be online for an initial build to fetch the library from a remote Maven location.
+.. warning:: If you choose to install a vendor dependency via the "online" mode, you must reconnect your computer to the internet and rebuild about every 30 days otherwise the cache will clear, completely deleting your cached library install.
 
-The JSON file allows specification of complex libraries with multiple components (Java, C++, JNI, etc.) and also helps handle some complexities related to simulation. Vendors choosing to provide a remote URL in the JSON also enable users to check for updates from within VS Code.
+How Does It Work?
+^^^^^^^^^^^^^^^^^
 
-.. note:: The vendor JSON files are actually processed by GradleRIO once they are in your projects ``vendordeps`` folder. If you are using another IDE, you will need to manually create a "vendordeps" folder in your project and copy any desired vendor JSON files from the "wpilib/YYYY" folder (where they should be placed by an offline installer) or download them directly from the vendor and place them into the folder in the project.
```

Comment:
Why is this being removed?

Reply:
I figure it would probably be much better to get users to use Gradle for other IDEs (from the Command-Line section) especially since we mention using the command line. If we wanted to keep this note for "manual"ly adding vendordeps, we could probably put it into another section like we do for the command line / VS Code ways